### ISSUE TITLE
Prisma generate is leaving a defunct node process behind (zombie)

### ISSUE THREAD

**Issue by bradrf** [prisma/prisma#17024]
### Bug description

After every run of `prisma generate` there's a new zombie node process left behind:
```
$ ps aux | grep defunct
node         616  0.0  0.0   3244   704 pts/0    SN+  22:24   0:00 grep defunct

$ ./node_modules/.bin/prisma generate
Prisma schema loaded from schema.prisma

✔ Generated Prisma Client (4.8.0 | library) to ./node_modules/@prisma/client in 129ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client

$ ps aux | grep defunct
node         846  7.5  0.0      0     0 ?        ZNs  22:24   0:00 [node] <defunct>
node         871  0.0  0.0   3244   640 pts/0    SN+  22:24   0:00 grep defunct
```

### How to reproduce

Execute `prisma generate`

### Expected behavior

No leftover child node processes remain.

### Prisma information

```prisma
datasource db {
  url      = env("DATABASE_URL")
  provider = "postgresql"
}

generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["extendedWhereUnique"]
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
}
```

### Environment & setup

- OS:  Docker container running Linux 6.0.12 (Debian "bullseye")
- Database: PostgreSQL 14.6 (Debian 14.6-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.
1 20210110, 64-bit
- Node.js version: v18.12.1

### Prisma Version

```
prisma                  : 4.8.0
@prisma/client          : 4.8.0
Current platform        : debian-openssl-1.1.x
Query Engine (Node-API) : libquery-engine d6e67a83f971b175a593ccc12e15c4a757f93ffe (at node_modules/@prism
a/engines/libquery_engine-debian-openssl-1.1.x.so.node)
Migration Engine        : migration-engine-cli d6e67a83f971b175a593ccc12e15c4a757f93ffe (at node_modules/@
prisma/engines/migration-engine-debian-openssl-1.1.x)
Introspection Engine    : introspection-core d6e67a83f971b175a593ccc12e15c4a757f93ffe (at node_modules/@pr
isma/engines/introspection-engine-debian-openssl-1.1.x)
Format Binary           : prisma-fmt d6e67a83f971b175a593ccc12e15c4a757f93ffe (at node_modules/@prisma/eng
ines/prisma-fmt-debian-openssl-1.1.x)
Format Wasm             : @prisma/prisma-fmt-wasm 4.8.0-61.d6e67a83f971b175a593ccc12e15c4a757f93ffe
Default Engines Hash    : d6e67a83f971b175a593ccc12e15c4a757f93ffe
Studio                  : 0.479.0
Preview Features        : extendedWhereUnique
```

**Comment by jkomyno** [comment]
Hi @bradrf, I realise this is an old issue, but this has been fixed for a long time, at least since Prisma 5.0.0.
We completely removed the binary Prisma Engine that took care of validating and formatting Prisma schema files, and replaced it with a Wasm implementation. This implies that no zombie processes are left in your system :)

**Comment by bradrf** [comment]
No worries, thanks for letting me know, @jkomyno!
